instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/SqsReceiveMessageRequestModel.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.awssdk.v1_11;
+
+import com.amazonaws.AmazonWebServiceRequest;
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Method;
+
+class SqsReceiveMessageRequestModel {
+
+  private final AmazonWebServiceRequest request;
+
+  SqsReceiveMessageRequestModel(final AmazonWebServiceRequest request) {
+    this.request = request;
+  }
+
+  void setSystemAttribute(String attribute) throws Throwable {
+    // using reflection as MethodHandle requires return type definition which is unavailable
+    Method withAttributeNames = request.getClass().getMethod("withAttributeNames", String[].class);
+    MethodHandles.publicLookup()
+        .unreflect(withAttributeNames)
+        .invokeWithArguments(request, attribute);
```

Comment:
But why do we need to use reflection? Can we use a compileOnly dependency?

Reply:
I spoke about this with @mateuszrzeszutek a my understanding is that if AWS SDK SQS classes are used directly, muzzle will disable whole AWS SDK 1.11 instrumentation if AWS SDK SQS module is not found in the runtime, ie for projects that still use AWS SDK but without SQS module (which can happen). LMK if this is correct :)

Reply:
That comment was for `camel-aws` (which is separate jar/library, not included in `camel-core`), I'm not sure whether this still holds for AWS SDK 1.11 (I promise I'll look at your PR today 😄 ).
But the general rule is: if you instrument a single jar library all instrumented versions have similar/same API then don't bother with reflection/handles.

Reply:
Hmm, it look like `aws-java-sdk-sqs` is a part of `aws-java-sdk` - since we aim to instrument the latter one there's probably no need to use handles here

Reply:
OK, great, failed to check that the SDK is already a dependency. Removed dynamic code then 🤷